BBDown/CommandLineInvoker.cs

```diff
@@ -35,6 +35,7 @@ internal static class CommandLineInvoker
     private static readonly Option<bool> SkipCover = new(["--skip-cover"], "跳过封面下载");
     private static readonly Option<bool> ForceHttp = new(["--force-http"], "下载音视频时强制使用HTTP协议替换HTTPS(默认开启)");
     private static readonly Option<bool> DownloadDanmaku = new(["--download-danmaku", "-dd"], "下载弹幕");
+    private static readonly Option<string> DownloadDanmakuFormat = new(["--download-danmaku-format", "-ddf", "指定需下载的弹幕格式"]);
```

Comment:
这个选项支持多个值，最好改成`--download-danmaku-formats`这种复数形式, 同时 `DownloadDanmakuFormat` -> `DownloadDanmakuFormats`。

然后描述信息可以详细一些，告诉用户程序默认下载两种，同时需要体现用户如何才能自定义想要的格式。

而且你似乎把description也写到alias里了（
```suggestion
    private static readonly Option<string> DownloadDanmakuFormats = new(["--download-danmaku-format", "-ddf"], "指定需下载的弹幕格式列表, 以英文逗号分隔(默认为: xml,ass)");
```